tests/wpt/web-platform-tests/css/css-images/support/generate-object-fit-and-position-canvas-tests.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
```

Comment:
Any changes under tests/wpt/web-platform-tests should be reverted; those files are vendored from https://github.com/web-platform-tests/wpt.

Reply:
Reverted everything under `tests/wpt/web-platform-tests`.